### PR TITLE
Feat/button icons

### PR DESCRIPTION
**Pull request by bladicito**

### CODE REVIEW

**Comment by lehmi11** [comment]
You need to rename the PR in order to make semantic release work ;-)

File: components/button/Button.tsx [hunk 1]
```diff
@@ -99,6 +109,28 @@ const Button = ({
         modifier = [...modifier, ...md.topContainer];
     }
 
+    if (kind === EButtonKinds.COPY_TO_CLIPBOARD) {
+      modifier = [
+        ..._.difference(modifier, [
```
Comment (lehmi11) [inline components/button/Button.tsx]: I have never worked with Loadash so far. Do you need it here to compare the values of the arrays and only add the different ones?

File: components/button/Button.tsx [hunk 2]
```diff
@@ -192,27 +237,68 @@ const Button = ({
...
+          onClick={(evt) => {
+            if (kind === EButtonKinds.COPY_TO_CLIPBOARD) {
+              if (clipboardData) {
+                console.log('copying data');
+                navigator.clipboard
+                  .writeText(clipboardData)
+                  .then(() => {
+                    setHighlighted(true);
+                    setLabelText(clipboardCopySuccessLabel);
+                    setTimeout(() => {
```
Comment (lehmi11) [inline components/button/Button.tsx]: nice implementation with setTimeout

**Review by lehmi11 — APPROVED** [review]
Good Job. LGTM 👍

File: components/button/Button.tsx (hunk 1 above)
Comment (bladicito) [reply components/button/Button.tsx]: yes, it is only intended to remove the items on the second array. Instead of making a filter with different cases.

**Pull request merged**